### ISSUE TITLE
Kubernetes and Docker commands

### ISSUE THREAD

**Issue by sabir-catalogic** [hash-define-organization/Hacktober-Fest-2021#229]
what about some useful command and tips for Kubernetes and Docker.
It will be accepted or not?

**Comment by WINOFFRG** [comment]
Yes, it will be good. You can list at least 10 commands of each. More will be better, with images and a brief definition.
